### PR TITLE
Portfolio mobile menu

### PR DESCRIPTION
**Pull request by ifeanyi305**
we just completed the mobile menu for my portfolio, project and  we added the following:
- Added Js file
- HTML,CSS and Js Linters checked
- Added a descriptive Readme file
- Added menu for the mobile version

### CODE REVIEW

**Review by ShadyShawkat — APPROVED** [review]
## Hi @boo-shehab, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted the wrong pull request link of your project to your milestone's submission. Kindly make sure to submit your PR link (while also pair-programming with your partner) of your project for this milestone.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -16,23 +16,23 @@
...
-                <ul class="nav-list">
-                  <li class="nav-item"><a href="#">portfolio</a></li>
-                  <li class="nav-item"><a href="#">About</a></li>
-                  <li class="nav-item"><a href="#">Contact</a></li>
-                </ul>
                 <div class="hambuger">
                   <div class="bar"></div>
                   <div class="bar"></div>
                   <div class="bar"></div>
                 </div>
```
Comment (prolajumokeoni) [inline index.html:23]: ### OPTIONAL


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Screenshot_479  | Screenshot_480


- [ ] Kindly align the logo and hamburger icon in your header to look like the Figma design 👍🏿

**Review by prolajumokeoni — APPROVED** [review]
Hi @ifeanyi305, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- @boo-shehab  Kindly make sure you implement the mobile menu feature on your own repository, this is a pair programming project  but based on the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#work-and-submission-mode) each partner needs to implement this feature on their own repository 👍🏿 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ifeanyi305) [reply index.html:23]: Thank you for your feedback! I will adjust the alignment of the logo and hamburger icon in the header to match the Figma design as closely as possible.

Thanks again for your help!

**Pull request merged**